employee/models.py

```
@@ -513,12 +516,41 @@ def get_subordinate_employees(self):
+            return
+
+        try:
+            file.seek(0)
+            content = file.read()
+        except Exception:
+            raise ValidationError({"employee_profile": "Unable to read uploaded file."})
+
+        is_svg = False
+        try:
+            text = content.decode("utf-8", errors="strict")
+            root = ET.fromstring(text)
```

Ruff (0.14.14)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)